<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>turnRatioIntervalDialog</class>
 <widget class="QDialog" name="turnRatioIntervalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>90</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>90</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="LabelBegin">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;From&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="QSpinBoxBeginHour">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximum">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="QSpinBoxBeginMinute">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="QSpinBoxBeginSecond">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LabelEnd">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;to&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="QSpinBoxEndHour">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximum">
        <number>24</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="QSpinBoxEndMinute">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="QSpinBoxEndSecond">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="LayoutButtons">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonReset">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/eNetEditorResources.qrc">
         <normaloff>:/main/reset.ico</normaloff>:/main/reset.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonAccept">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Accept</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/eNetEditorResources.qrc">
         <normaloff>:/main/accept.ico</normaloff>:/main/accept.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonCancel">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/eNetEditorResources.qrc">
         <normaloff>:/main/cancel.ico</normaloff>:/main/cancel.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../resources/eNetEditorResources.qrc"/>
 </resources>
 <connections/>
</ui>
